minigalaxy/ui/library.py

```diff
@@ -193,6 +193,15 @@ def __get_installed_games(self) -> List[Game]:
                 games.append(Game(name=name, game_id=game_id, install_dir=full_path, category=category))
             else:
                 games.extend(get_installed_windows_games(full_path, game_categories_dict))
+
+        # try to repair a corrupted list of ongoing downloads
+        # if something is considered 'installed', it shouldn't be on the download list anymore
+        ongoing_downloads = self.config.current_downloads()
+        for game in games:
+            if game.id in ongoing_downloads:
+                ongoing_downloads.remove(game.id)
+        self.config.current_downloads = ongoing_downloads
```

Comment:
This is probably nicer
```suggestion
        for game in games:
            if game.id in self.config.current_downloads:
                self.config.current_downloads.remove(game.id)
```

Reply:
As written above, config internally does write the changes when `config.current_downloads` is assigned. You're dropping the assignment, which would mean that the change will not be persisted.

An alternative would be to use `config.remove_ongoing_download` which does this internally. But this method was meant for singular updates from games, doing it in a loop would re-write the entire minigalaxy config file in each iteration.